### PR TITLE
Fix recent constant change affecting named enums

### PR DESCRIPTION
**Pull request by rick-nguyen**
Previous PR for named formulas would mark named enums as false even if they were true: https://github.com/microsoft/Power-Fx/pull/1602

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs [hunk 1]
```diff
@@ -2807,13 +2806,14 @@ public override void Visit(FirstNameNode node)
                     return;
                 }
 
+                var isConstantNamedFormula = lookupInfo.Kind == BindKind.Enum;
```
Comment (lesaltzm) [inline src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs:2809]: naming: this is just `isConstant`. An enum is not a named formula.

Comment (lesaltzm) [reply src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs:2809]: Or, just set this to false, since you have the enum check lower down.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**